src/pages/signin/SignInPage.js

```diff
@@ -102,7 +104,7 @@ class SignInPage extends Component {
         }
 
         return (
-            <SafeAreaView style={[styles.signInPage]}>
+            <View style={[styles.signInPage, StyleUtils.getSafeAreaPadding(this.props.insets, 1)]}>
```

Comment:
For some reason SafeAreaView flickers with the wrong insets on Android, so use insets from withSafeAreaInsets which are fine.

I had to add an extra parameter to `StyleUtils.getSafeAreaPadding` since by default it multiplies bottom and side insets by 0.7 which we don't want here or the size we calculate in `SignInPageLayout` will be wrong

Reply:
Can we plz include a code comment explaining this and linking to the issue?